.github/workflows/validate-nest-prisma-inhouse.yml

```diff
@@ -60,8 +81,8 @@ jobs:
           # [info,warning,error].
           # More about reviewdog reporter level at
           # https://github.com/reviewdog/reviewdog#reporters
-          level: warning
-          workdir: ./src
+          level: ${{inputs.level}}
+          workdir: ${{ inputs.directory }}/src
```

Comment:
Are we absolutely sure that all of our `Nest` projects have `/src` ? Isn't there a way how we can get/set this directory automatically rather then hardcoding it?